packages/p2p/src/stores/general-store.js

```diff
@@ -30,6 +30,9 @@ export default class GeneralStore extends BaseStore {
     @observable should_show_popup = false;
     @observable user_blocked_until = null;
     @observable is_high_risk_fully_authed_without_fa = false;
+    @observable banner_config = {
```

Comment:
Setting a variable as an `observable` here means that you `want to track any changes and re-render`. Treating it as a constant here is not needed